exonum-java-binding/testkit/src/test/java/com/exonum/binding/testkit/TestKitTest.java

```diff
@@ -286,30 +276,12 @@ private void checkTestServiceInitialization(TestKit testKit, String serviceName,
   private void checkTestService2Initialization(TestKit testKit, String serviceName, int serviceId) {
     // Check that service appears in dispatcher schema
     checkIfServiceEnabled(testKit, serviceName, serviceId);
-    TestService2 service = testKit.getService(serviceName, TestService2.class);
-    // Check that TestService2 API is mounted
-    Node serviceNode = service.getNode();
-    EmulatedNode emulatedTestKitNode = testKit.getEmulatedNode();
-    assertThat(serviceNode.getPublicKey())
```

Comment:
Should we retrieve public key from core schema instead and validate them against `emulatedTestKitNode` public key or is that unnecessary?

Reply:
We do, but not using core schema (we just check that the node type (validator or auditor) is correct and keys are not null). Should I put this PR on WIP until https://github.com/exonum/exonum-java-binding/pull/1185 is merged and then add the full test?

Reply:
Created a test for that, need a merged #1185 though